db/scripts/nz-buildings-load.in

```diff
@@ -101,6 +101,20 @@ psql_exit() {
     fi
 }
 
+# create test admin_bdys schema
+
+if [ "${WITH_TEST_DATA}" = true ]; then
+    psql -c "DROP SCHEMA IF EXISTS admin_bdys CASCADE;"
+    echo ${SCRIPTSDIR}/tests/testdata/09-create_test_admin_bdys_schema.sql >&2
+    psql -f ${SCRIPTSDIR}/tests/testdata/09-create_test_admin_bdys_schema.sql $ON_ERROR_STOP
+fi
+
+if [ "${WITH_PLUGIN_SETUP}" = true ]; then
```

Comment:
I agree but the admin_bdys schema needs to be made before we run lines 131-138 because the new functions for the suburb table use the admin_bdys.nz_locality table.